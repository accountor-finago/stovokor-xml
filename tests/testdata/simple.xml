<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Document>
	<foo>
		<!-- Some data to be replaced -->
		<author>
			<name>Very Secret Name</name>
			<personalId>123456</personalId>
		</author>
		<customers>
			<customer>
				<name>Jean-Luc Picard</name>
				<personalId>1234567</personalId>
				<bankAccount>FI10 4725 0961 0005 73</bankAccount>
				<motto>To boldly go, where no one has gone before</motto>
			</customer>
			<customer>
				<name>Worf</name>
				<personalId>1234568</personalId>
				<bankAccount>FI3647763216000644</bankAccount>
				<motto>Qapla'</motto>
			</customer>
		</customers>
		<!-- Some data to be kept -->
		<bar>
			<doNotChange>Some values, which should not be changed</doNotChange>
			<localizations>
				<!-- To check, that special chars will not be broken -->
				<localization lang="FI">Joku hyvin tärkeä teksti suomeksi</localization>
				<localization lang="PL">Jakiś bardzo ważny tekst po polsku</localization>
			</localizations>
		</bar>
	</foo>
</Document>