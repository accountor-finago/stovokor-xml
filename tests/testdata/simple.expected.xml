<?xml version='1.0' encoding='utf-8'?>
<Document>
	<foo>
		<!-- Some data to be replaced -->
		<author>
			<!--Obfuscated-->
			<name>Generator mock value for: name_regenerate Very Secret Name</name>
			<!--Obfuscated-->
		<personalId>Generator mock value for: num -l 6</personalId>
		</author>
		<customers>
			<customer>
				<!--Obfuscated-->
				<name>Generator mock value for: name_regenerate Jean-Luc Picard</name>
				<!--Obfuscated-->
				<personalId>Generator mock value for: num -l 7</personalId>
				<!--Obfuscated-->
				<bankAccount>Generator mock value for: iban_regenerate FI10 4725 0961 0005 73</bankAccount>
				<motto>To boldly go, where no one has gone before</motto>
			</customer>
			<customer>
				<!--Obfuscated-->
				<name>Generator mock value for: name_regenerate Worf</name>
				<!--Obfuscated-->
				<personalId>Generator mock value for: num -l 7</personalId>
				<!--Obfuscated-->
				<bankAccount>Generator mock value for: iban_regenerate FI3647763216000644</bankAccount>
				<motto>Qapla'</motto>
			</customer>
		</customers>
		<!-- Some data to be kept -->
		<bar>
			<doNotChange>Some values, which should not be changed</doNotChange>
			<localizations>
				<!-- To check, that special chars will not be broken -->
				<localization lang="FI">Joku hyvin tärkeä teksti suomeksi</localization>
				<localization lang="PL">Jakiś bardzo ważny tekst po polsku</localization>
			</localizations>
		</bar>
	</foo>
</Document>